docs/learn-DOT.md

```diff
@@ -28,20 +27,32 @@ Currently, DOTs have 15 decimal places, while the smallest unit is called the Pl
 | --- | --- |
 Planck| 1
 Point | 3
-Microdot (UDOT)| 6
-Millidot (MDOT)| 9
+Microdot (uDOT)| 6
+Millidot (mDOT)| 9
 Dot (DOT) | 12
 Blob | 15
 
+**Kusama**
+|Unit |Decimal Places|
+| --- | --- |
+Planck| 1
+Point | 3
+MicroKSM (uKSM)| 6
+MilliKSM (mKSM)| 9
+KSM (DOT) | 12
```

Comment:
I don't think so, since we lost those three digits of precision.

@lsaether since you talked with Gav and modified the chainspec, any thoughts here?

Reply:
Logan is out until Thursday, so I'm going to spin double-checking this off into another issue and assign it to him.  I'm pretty sure it's right but want to run it past him, and I want this to get in because of all the minor fixes (if this PR sits out there, going to get merge conflicts from other things)